modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -0,0 +1,142 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="overriding-default-node-ip-selection-logic_{context}"]
+= Optional: Overriding the default node IP selection logic 
+
+In most circumstances, the `NODEIP_HINT` is the API VIP. The IP selection logic will determine which subnet the `NODEIP_HINT` belongs to, and from that determine the interface in that subnet and select an IP on that interface. For example, a node that has two interfaces, `eth0` with the subnet range `10.0.0.10/24`, and `eth1` with the subnet range `192.0.2.5/24`, and with a default route via the `10.0.0.10` source address on `eth0`:
+
+.Example
+
+[source,terminal]
+----
+$ ip route show
```

Comment:
This shouldn't be necessary. I discuss this a bit more below, but CIDRs have to be provided in install-config so the deployer should already know what IP ranges are available. Basically this information is a prereq for doing a deployment at all, we don't need to cover it explicitly in this procedure.